frontend/src/pages/ViewArticle.js

```diff
@@ -1,20 +1,77 @@
-import { useContext, useEffect } from "react";
+import { useContext, useEffect, useState } from "react";
 import { Box } from "@mui/material";
 import { CurrentUrlContext } from "../context/CurrentUrlContext";
+import Dropdown from "../components/Dropdown";
+import ArticleTable from "../components/Table";
+
+import { getPractices } from "../services/practicesService";
+import { getArticle } from "../services/articlesService";
+import tablecolumns from "../components/tableColumns";
 
 const ViewArticle = () => {
-    // Current URL state
-    const [, setSelectedUrl] = useContext(CurrentUrlContext);
-
-    useEffect(() => {
-        setSelectedUrl("view-article");
-    }, [setSelectedUrl]);
-
-    return (
-        <Box sx={{ bgcolor: "#fff", margin: "12px", padding: "8px" }}>
-            <h1>View Articles</h1>
-        </Box>
-    );
+  // Current URL state
+  const [, setSelectedUrl] = useContext(CurrentUrlContext);
+
+  // Dropdown States
+  const [practices, setPractices] = useState([]);
+  const [selectedPractice, setSelectedPractice] = useState("");
+
+  // Articles
+  const [articles, setArticles] = useState([]);
+
+  // Loading State
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setSelectedUrl("view-article");
+  }, [setSelectedUrl]);
```

Comment:
Can remove this useEffect and combine it with the other one. Make sure to set url to /view-articles not /propose-articles. This state is being watched by the nav bar so it knows which menu item to highlight.

```suggestion
```